### ISSUE TITLE
module 'pcl' has no attribute 'load_XYZI'

### ISSUE THREAD

**Issue by yzfneo** [kcg2015/lidar_ground_plane_and_obstacles_detections#1]
I follow the steps to install python-pcl and required dependencies, but when I run the scripts from lidar_segementation_detection.ipynb, this error keeps poping. Seems pcl module is not properly installled...Any advice is greatly appreciated.

module 'pcl' has no attribute 'load_XYZI'

**Comment by kcg2015** [comment]
@yzfneo, I also had similary issues with Python PCL. But somehow it worked for a while when I coded for this repo. Could you let me know the version of your OS, Python, Cython, etc?

**Comment by yzfneo** [comment]
@kcg2015 Seems that I fixed this issue myself...
For backgroud information, I configured this on google colab.
Previously I first installed PCL, and python-pcl simply by "apt install". And even if I reinstall everything following every steps from python-pcl, it seems that the pcl module being imported is the one installed using "apt install".
So my accidental fix for this is, 
1) Install PCL module manually. (sudo apt-get install libpcl-dev -y)
2) make a clone of  [python-pcl](https://github.com/strawlab/python-pcl) and import pcl from that directory.
Done.
